jwt-authorizer/src/authorizer.rs

```diff
@@ -38,6 +41,7 @@ where
     pub key_source: KeySource,
     pub claims_checker: Option<FnClaimsChecker<C>>,
     pub validation: crate::validation::Validation,
+    pub jwt_source: JwtSource,
```

Comment:
It seems strange to me that the JwtSource is in the authorizer; My expected usage would be for the service to validate JWT tokens against different keys, however I'm not sure why different authorizer would have different ways of passing the token to the service. That mean potentially a client would need to pass the tokens differently depending on how it got the JWT?

Reply:
This is certainly not a frequent use, but all the configuration is in JwtAuthorizer,
you need to take the jwt_source of one of the authorizers but from witch one? Choosing the first and apply it to all?
More over we could imagine a use case where we have more than one possibility of passing the token (depending on the client), then we can add an authorizer per jwt_source.